uniffi/src/lib.rs

```diff
@@ -205,11 +206,7 @@ pub fn run_bindgen_helper(component_name: Option<&str>) -> Result<()> {
                 .arg(
                     clap::Arg::with_name("lib_file")
                         .takes_value(true)
-                        .required(if let Some(_) = default_lib_file {
-                            false
-                        } else {
-                            true
-                        })
+                        .required(default_lib_file.is_some())
```

Comment:
```suggestion
                        .required(default_lib_file.is_none())
```

Realized while adding the tests for strings that I had to manually set the lib_file when running `generate` 😄

Reply:
nice catch, thanks! 😅